diff --git a/Dockerfile b/Dockerfile
--- a/Dockerfile
+++ b/Dockerfile
@@ -113,7 +113,7 @@
 # node service requirements
 # netbase is required by rpcbind/rpcinfo to work properly
 # /etc/{services,rpc} are required
-RUN apt-get update && \
+RUN apt-get update && apt-get upgrade -y --no-install-recommends && \
   apt-get install -y wget netbase zip bzip2 socat e2fsprogs exfatprogs xfsprogs btrfs-progs fatresize dosfstools ntfs-3g nfs-common cifs-utils fdisk gdisk cloud-guest-utils sudo rsync procps util-linux nvme-cli fuse3 && \
   rm -rf /var/lib/apt/lists/*
 
