diff --git a/Dockerfile b/Dockerfile
--- a/Dockerfile
+++ b/Dockerfile
@@ -28,13 +28,14 @@
 
 # build
 ENV CGO_ENABLED=0
-RUN xx-go build -trimpath -a -o notification-controller main.go
+RUN go get google.golang.org/grpc@v1.83.1 golang.org/x/crypto@v0.56.0 golang.org/x/net@v0.57.0 golang.org/x/text@v0.41.0 \
+    && xx-go build -trimpath -a -o notification-controller main.go
 
 FROM alpine:3.23
 
 ARG TARGETPLATFORM
 
-RUN apk --no-cache add ca-certificates \
+RUN apk --no-cache upgrade && apk --no-cache add ca-certificates \
   && update-ca-certificates
 
 COPY --from=builder /workspace/notification-controller /usr/local/bin/
